### PR TITLE
Add support for periodic tasks

### PR DESCRIPTION
**Pull request by bakura10**
This PR adds support for Elastic Beanstalk periodic tasks, so that they are protected for localhost and work the exact same way as for standard task.

### CODE REVIEW

File: src/Middleware/WorkerMiddleware.php [hunk 1]
```diff
@@ -97,7 +119,33 @@ public function __invoke(
         /** @var ResponseInterface $response */
         $response = $pipeline($request, $response);
 
-        return $response->withHeader('X-HANDLED-BY', 'ZfrEbWorker');
+        return $response->withHeader('X-Handled-By', 'ZfrEbWorker');
```
Comment (bakura10) [inline src/Middleware/WorkerMiddleware.php:110]: Just for consistency with how uppercasing is done by AWS. Should not cause any BC as if I remember getHeader is supposed to be case-insensitive.

File: src/Middleware/WorkerMiddleware.php [hunk 2]
```diff
@@ -79,10 +79,20 @@ public function __invoke(
...
+        // the worker daemon automatically adds the "X-Aws-Sqsd-Taskname" header. When we find it, we simply use this
+        // name as the message name and continue the process
+
+        if ($request->hasHeader('X-Aws-Sqsd-Taskname')) {
+            // The full message is set as part of the body
+            $name    = $request->getHeaderLine('X-Aws-Sqsd-Taskname');
+            $payload = [];
+        } else {
+            // The full message is set as part of the body
+            $body    = json_decode($request->getBody(), true);
```
Comment (danizord) [inline src/Middleware/WorkerMiddleware.php:92]: Could we rely on `$request->getParsedBody();` here?

**Review by danizord — APPROVED** [review]

File: src/Middleware/WorkerMiddleware.php (hunk 2 above)
Comment (bakura10) [reply src/Middleware/WorkerMiddleware.php:92]: I suppose so :p.

Comment (danizord) [reply src/Middleware/WorkerMiddleware.php:92]: Mhmm nvm, that assumes `BodyParamsMiddleware` is piped before.

**Pull request merged**